t/t.go

```diff
@@ -172,7 +175,9 @@ func stopCluster(composeFile, prefix string, wg *sync.WaitGroup, err error) {
 		if err != nil {
 			outputLogs(prefix)
 		}
-		cmd := command("docker-compose", "-f", composeFile, "-p", prefix, "down", "-v")
+		//cmd := command("docker-compose", "-f", composeFile, "-p", prefix, "down", "-v")
+		cmd := command("docker", "ps", "-aq", "|", "xargs", "docker", "stop", "|", "xargs", "docker", "rm")
```

Comment:
this was indeed a nice workaround - just curious, did this change from `$()` to `|` work?